src/services/near.js

```diff
@@ -42,9 +43,10 @@ export const getMemeComments = (meme) => {
     .account()
     .viewFunction(memeContractId, "get_recent_comments", {});
 };
+
 // a service to add a comment to a meme
 export const addComment = ({ meme, text }) => {
-  const memeContractId = meme + "." + CONTRACT_ID;
+  const memeContractId = `${meme}.${CONTRACT_ID}`;
   return wallet.account().functionCall({
     contractId: memeContractId,
     methodName: "add_comment",
```

Comment:
question (blocking): what happens if a request fails? I do not see any error handling (I have only checked this PR not the entire code thought).

Reply:
True, there is no error handling yet.

Reply:
done, added error handling.